_posts/2023-07-30-Server-Metadata.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Adding metadata to Servers"
+lang: "en"
+heading: "Metadata for servers"
+---
+
+To provide information about Servers or clarify policies for Bots, server owners can add metadata to their servers' welcome message. This page gives a rough format about how the metadata could look like.
```

Comment:
```suggestion
To provide information about Servers and specify policies for Bots, a server owner can add metadata to their server welcome message. This page describes metadata a welcome message might include.
```

Can't decide if _might_ makes sense. Want to communicate that it's not required, and it's not limited to these. I guess I wonder which are supported in any way. Admin contact, for example: Unless we surface this in some way, perhaps it's unwise to invite people to include it. Admin contact is the kind of data Jamulus Explorer might include. Until a tag is supported in some way, I wouldn't even want to mention it. So for example, I'd roll out support for a bot standard before rolling out this new page.

Reply:
We are somewhat compelled here to define bots. While people run stationary bots on their own servers, this is more about bots that connect to public servers ("roaming bots"). I'm only aware of the two I'm running. I'd call one a _snippet bot_, and the other is a _streaming bot_, but I haven't thought of a word to describe both, except _listening bot_, which I find uncomfortable. At this time there isn't a recording bot; establishing this featureset will improve my ability to provide one. We haven't quite established a v1 featureset... maybe the first release can just be YES/NO, and future releases can specify roaming bot policy with greater granularity.

Reply:
I mean the site needs to be very general. 

> I guess I wonder which are supported in any way

Anything which fits into the metadata box in theory. But that's up to the people using the metadata. 

I wouldn't add a might - but I don't feel too strong about it.

> Unless we surface this in some way, perhaps it's unwise to invite people to include it.

People could still include their home address, coordinates or even social media profile in there. But yes, normally you'd add an email or website.

Reply:
> So for example, I'd roll out support for a bot standard before rolling out this new page.

Still we should have a standard somewhere - at least a very general one people can follow.

Reply:
> maybe the first release can just be YES/NO, and future releases can specify roaming bot policy with greater granularity.

Yes sure. Then we'd drop the thing with the bot policy all together and just talk about metadata in general?

Reply:
I think now that metadata for bot policy is basically the only solid case for metadata. If you want to reach users, use clear text html. Bot policy reaches bot providers.